src/DssSpell.sol

```diff
@@ -25,14 +25,11 @@ import { DssSpellCollateralOnboardingAction } from "./DssSpellCollateralOnboardi
 contract DssSpellAction is DssAction, DssSpellCollateralOnboardingAction {
     // Provides a descriptive tag for bot consumption
     // This should be modified weekly to provide a summary of the actions
-    // Hash: seth keccak -- "$(wget https://raw.githubusercontent.com/makerdao/community/c421b8a52031169aa55170b752206250ab2d15cf/governance/votes/Executive%20Vote%20-%20January%2021%2C%202022.md -q -O - 2>/dev/null)"
+    // Hash: seth keccak -- "$(wget https://raw.githubusercontent.com/makerdao/community/5a8f09353e2a7d7f0d9f6a357cd05901348fe227/governance/votes/Executive%20vote%20-%20January%2024%2C%202022.md -q -O - 2>/dev/null)"
     string public constant override description =
-        "2022-01-21 MakerDAO Executive Spell | Hash: 0x741fefda04e07fbdcceb7b5e3f9704694f353771154380b3fd13b52ead67837d";
+        "2022-01-24 MakerDAO Emergency Executive Spell | Hash: 0x75da13d2bed7612e2e1960e7fe193453b66ead6e5d68b9a553258b0d421a21db";
 
-    // --- Rates ---
-    uint256 constant ZERO_FIVE_PCT_RATE      = 1000000000158153903837946257;
-    uint256 constant TWO_FIVE_PCT_RATE       = 1000000000782997609082909351;
-    uint256 constant THREE_PCT_RATE          = 1000000000937303470807876289;
+    address constant SB_LERP = 0x0239311B645A8EF91Dc899471497732A1085BA8b;
```

Comment:
silly question as I see it files to the vow, but where was this address taken from? can't find it in the archive/repo.

Reply:
I had to reach into the `LERP_FAB` to get it. It's currently the only tracked lerp.